test/usecases/net_send/simulate.py

```diff
@@ -3,23 +3,51 @@
 from neuron import h, gui
 from neuron.units import ms
 
-nseg = 1
 
-s = h.Section()
-s.nseg = nseg
+def simulate():
+    nseg = 1
 
-toggle = h.toggle(s(0.5))
+    s = h.Section()
+    s.nseg = nseg
 
-y_hoc = h.Vector().record(toggle._ref_y)
-t_hoc = h.Vector().record(h._ref_t)
+    toggle = h.toggle(s(0.5))
+    art_toggle = h.art_toggle()
 
-h.stdinit()
-h.tstop = 5.0 * ms
-h.run()
+    t_hoc = h.Vector().record(h._ref_t)
+    y_hoc = h.Vector().record(toggle._ref_y)
+    art_y_hoc = h.Vector().record(art_toggle._ref_y)
 
-t = np.array(t_hoc.as_numpy())
-y = np.array(y_hoc.as_numpy())
+    h.stdinit()
+    h.tstop = 5.0 * ms
+    h.run()
 
-y_exact = np.array(t >= 2.0, np.float64)
+    t = np.array(t_hoc.as_numpy())
+    y = np.array(y_hoc.as_numpy())
+    art_y = np.array(art_y_hoc.as_numpy())
 
-assert np.all(np.abs(y - y_exact) == 0), f"{y} != {y_exact}, delta: {y - y_exact}"
+    return t, y, art_y
+
+
+def check_solution(t, y, art_y):
+    y_exact = np.array(t >= 2.0, np.float64)
+    art_y_exact = np.array(t >= 1.5, np.float64)
```

Comment:
What do the two numbers here (2.0 and 1.5) represent? Are they related to some of the values used in the mod file?

Reply:
I've added a brief description to the top. The two numbers are the time at which the event should be received. At that point `y` toggles "on".

Reply:
Ahh, so the time is basically the first argument to `net_send` in the mod file?

Reply:
Correct, though technically, its the delay from the current time.